std/assembly/index.d.ts

```diff
@@ -1467,6 +1467,8 @@ interface INativeMath<T> extends IMath<T> {
   sincos_cos: T;
   /** Seeds the random number generator. */
   seedRandom(value: i64): void;
+  /** Multiplies a floating point `x` by 2 raised to power exp `n`. */
+  scalbn(x: T, n: i32): T;
```

Comment:
Is this useful to a user as a public API?

Reply:
I guess yes. It's has different name in C/C++ as `frexp`